front-end-tools/package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "front-end-tools",
+    "packageManager": "yarn@3.2.0",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "engines": {
+        "node": ">=16.x"
+    },
+    "dependencies": {
+        "@concordium/browser-wallet-api-helpers": "^2.5.0",
+        "@concordium/react-components": "^0.3.0",
+        "@concordium/wallet-connectors": "^0.3.1",
```

Comment:
All symbols from `wallet-connectors` are re-exposed in `react-components` so there's no need to explicitly depend on it (just make sure that you're imports refer to `react-components` only).